src/tree/values/nodes/Whitespace.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Represents a piece of whitespace that cannot be represented by any other structured element.
+ */
+class Whitespace {
+	/**
+	 * Represents a piece of whitespace that cannot be represented by any other structured element.
+	 *
+	 * @returns {void}
+	 */
+	constructor() {
```

Comment:
I checked with @atimmer , and this element should be able to represent any whitespace (e.g. `"\t"`, `"       "`, `\n`).
I would add a parameter to the constructor in which you can add the needed whitespace and change the constructor accordingly.